datagrid_gtk3/tests/utils/test_transformations.py

```diff
@@ -0,0 +1,38 @@
+"""Data transformation utilities test cases."""
+
+import unittest
+
+from datagrid_gtk3.utils.transformations import degree_decimal_str_transform
+
+
+class DegreeDecimalStrTransformTest(unittest.TestCase):
```

Comment:
I was surprised not to see any other test case for the transformations. That explains everything.

My personal preference is to have the testing code in modules named after the original source code prefixes with `test_`, since that makes test cases easy to find. Unless you have some concern, I'd suggest to merge this PR as it is, and then work on moving the other test cases in a different PR.